### ISSUE TITLE
[Feature Request] What about bring translate function to Schildi?

### ISSUE THREAD

**Issue by alrcatraz** [SchildiChat/schildichat-graphics#7]
Greetings!

I **really** love SchildiChat, it makes me possible to customise the interface, making it more suitable for me!

I have used a series of chatting apps and social apps, and I found it really important to translate text (and if possible, even text in pictures) immediately, by - for example - a "translate" button. Cannot translate (and sometimes even copy text) make the experience chatting with people using different languages terrible, thus I really recommend to bring Schildi this function!

Hope my advice can make Schildi better!

**Comment by SpiritCroc** [comment]
Some more discussion on that topic: https://github.com/SchildiChat/SchildiChat-android/issues/147
tl;dr We won't work on this, but if someone comes up with a privacy-preserving FOSS solution, we may be open for contributions (but better keep in touch to get feedback early).

**Comment by alrcatraz** [comment]
I really understand! Thanks!
